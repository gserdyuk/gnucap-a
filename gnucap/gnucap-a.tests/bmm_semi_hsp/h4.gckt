'trunc error check
.param a=1 b=2
V1   1  0  ac a dc b pwl 0 0 1p 1
r1   1  0 50k

.list

.print op v(1) i(v1)
.op

.print ac v(1) i(v1)
.ac dec 2 1 10

.end
