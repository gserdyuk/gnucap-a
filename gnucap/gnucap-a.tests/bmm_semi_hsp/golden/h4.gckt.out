Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
'trunc error check 
V1 ( 1 0 ) pwl( 0., 0.  1.p, 1. )  DC b  AC a
r1 ( 1 0 )  50.K
#           v(1)       i(V1)     
 27.        2.        -40.u      
#Freq       v(1)       i(V1)     
 1.         1.         20.u      
 3.1623     1.         20.u      
 10.        1.         20.u      
